happi/item.py

```diff
@@ -59,8 +59,7 @@ class EntryInfo:
     ------
     .. code::
 
-        class MyDevice(Device):
-
+        class MyItem(HappiItem):
             my_field = EntryInfo('My generated field')
             number   = EntryInfo('Device number', enforce=int, default=0,
                                  enforce_doc='This must be a number')
```

Comment:
Unrelated to this file: `loader.py` has some arguments confusingly named `device` that are typed as `HappiItem`

Reply:
I missed that entire file, so it seems. A big commit here that needs a careful eye: [0c813f3](https://github.com/pcdshub/happi/pull/266/commits/0c813f3585b3dc11657f6ca1afaf716478daa1c4)